src/app/Http/Middleware/BackpackErrorViews.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Str;
+
+class BackpackErrorViews
```

Comment:
All our other middleware have verbs in the names, they're action-named. So how about we name this middleware something like... `UseBackpackErrorViews` or `UseBackpackErrorViewsInAdminPanel` or `UseBackpackErrorViewsInAdminRoutes`